apps/web/utils/supabase/client.ts

```
@@ -3,8 +3,85 @@
+                  });
+                }
+                return {
+                  data: { subscription: { unsubscribe: () => {} } },
+                };
+              },
+              signOut: async () => ({ error: null }),
+            },
+            from: (table: string) => ({
+              select: () => ({
+                single: async () => ({ data: {}, error: null }),
+                then: async () => [],
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)